plugins/pencilblue/controllers/actions/forgot_password.js

```diff
@@ -43,7 +43,7 @@ module.exports = function ForgotPasswordControllerModule(pb) {
         var ctx = this.getServiceContext();
         ctx.userService = this.userService;
         ctx.siteService = new SiteService(this.getServiceContext());
-        ctx.emailService = new EmailService(this.getServiceContext());
+        ctx.emailService = new EmailService({site: ctx.site});
```

Comment:
This was changed to the global email settings are used and not the site specific ones. I can just as well make a copy of the context and set `onlyThisSite` to false (I think)